modules/roles_profiles/manifests/profiles/windows_datacenter_generic_worker_14_1_2.pp

```diff
@@ -58,7 +58,7 @@
             class{ 'win_generic_worker::hw_config':
                 taskcluster_access_token => $taskcluster_access_token,
                 worker_type              => $facts['custom_win_gw_workertype'],
-                client_id                => 'project/releng/generic-worker/gecko-t-win10-64-hw/production',
+                client_id                => "project/releng/generic-worker/datacenter-${facts['custom_win_gw_workertype']}",
```

Comment:
I am not sure why this is showing up in the diff. This matches what is currently in the manifests.